elliott/elliottlib/cli/konflux_release_cli.py

```
@@ -178,13 +187,103 @@ def get_object_name(self) -> str:
+        # Fetch the ReleasePlanAdmission
+        rpa_data = await self._fetch_release_plan_admission(release_plan)
+
+        # Extract allowed components from data.mapping.components
+        try:
+            mapping = rpa_data.get("data", {}).get("mapping", {})
+            allowed_components: List[dict] = mapping.get("components", [])
+        except (AttributeError, TypeError):
+            raise ValueError(
+                f"ReleasePlanAdmission '{release_plan}' has invalid structure. "
+                "Expected 'data.mapping.components' to be a list."
+            )
```

Ruff (0.15.7)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)